frontend/src/app/settings/modelsources/t4c-settings/edit-t4c-instance/edit-t4c-instance.component.ts

```diff
@@ -47,7 +52,6 @@ export class CreateT4cInstanceComponent implements OnInit {
     host: new FormControl('', Validators.required),
     port: new FormControl(null as number | null, [
       Validators.required,
-      Validators.pattern(/[\d^0]\d*/),
```

Comment:
This validator was useful to assert that the port did not start with 0, to prevent for example a port looking like 01337.

Reply:
I also checked, without this decorator, decimal number (that are no valid port) are accepted.

Reply:
Yes, decimal numbers should be permitted of cause. I will change the validator pattern to `/^\d*$/`.